content/en/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -15,196 +15,71 @@ This page is an overview of Kubernetes.
+* Resource isolation: Predictable application performance.
+* Resource utilization: High efficiency and density.
 
 ## Why do I need Kubernetes and what can it do
 
-Kubernetes has a number of features. It can be thought of as:
-
-- a container platform
-- a microservices platform
-- a portable cloud platform
-and a lot more.
-
-Kubernetes provides a **container-centric** management environment. It
-orchestrates computing, networking, and storage infrastructure on
-behalf of user workloads. This provides much of the simplicity of
-Platform as a Service (PaaS) with the flexibility of Infrastructure as
-a Service (IaaS), and enables portability across infrastructure
-providers.
-
-## How Kubernetes is a platform
-
-Even though Kubernetes provides a lot of functionality, there are
-always new scenarios that would benefit from new
-features. Application-specific workflows can be streamlined to
-accelerate developer velocity. Ad hoc orchestration that is acceptable
-initially often requires robust automation at scale. This is why
-Kubernetes was also designed to serve as a platform for building an
-ecosystem of components and tools to make it easier to deploy, scale,
-and manage applications.
-
-[Labels](/docs/concepts/overview/working-with-objects/labels/) empower
-users to organize their resources however they
-please. [Annotations](/docs/concepts/overview/working-with-objects/annotations/)
-enable users to decorate resources with custom information to
-facilitate their workflows and provide an easy way for management
-tools to checkpoint state.
-
-Additionally, the [Kubernetes control
-plane](/docs/concepts/overview/components/) is built upon the same
-[APIs](/docs/reference/using-api/api-overview/) that are available to developers
-and users. Users can write their own controllers, such as
-[schedulers](https://github.com/kubernetes/community/blob/{{< param "githubbranch" >}}/contributors/devel/scheduler.md),
-with [their own
-APIs](/docs/concepts/api-extension/custom-resources/)
-that can be targeted by a general-purpose [command-line
-tool](/docs/user-guide/kubectl-overview/).
-
-This
-[design](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md)
-has enabled a number of other systems to build atop Kubernetes.
+Containers are a good way to bundle and run your applications. In a production environment, you need to manage the containers that run the applications and ensure that there is no downtime. For example, if a container goes down another container needs to restart. Wouldn't it be easier if this behavior was handled by a system?
+
+That's how Kubernetes comes to the rescue! Kubernetes provides you with a framework to run distributed system resiliently. It takes care of your scaling requirements, failover, deployment patterns, and more. For example, Kubernetes can easily manage a canary deployment for your system.
+
+Kubernetes provides you with:
+* Service discovery and load balancing
```

Comment:
list not rendering well: https://deploy-preview-14657--kubernetes-io-master-staging.netlify.com/docs/concepts/overview/what-is-kubernetes/#why-do-i-need-kubernetes-and-what-can-it-do